app/controllers/users_controller.rb

```diff
@@ -0,0 +1,5 @@
+class UsersController < ApplicationController
+  def show
+    @user = User.find(params[:id])
+  end
+end
```

Comment:
- [ ] The Users controller should contain the `index` method.

Reply:
Added the method as instructed